modules/docs/arrow-docs/docs/docs/effects/fx/polymorphism/README.md

```diff
@@ -158,10 +157,10 @@ The following combinators illustrate how the Functor hierarchy functions are poi
 | MonadDefer.delay    | `IO.delay { 1 }` | `effect { 1 }` |
 | MonadDefer.defer    | `IO.defer { IO { 1 } }` | `effect { 1 }` |
 
-This is in general true for effectful data types that are non-commutative. 
-Note that implicit CPS style with non-blocking direct binding has the disadvantage that for non-commutative monads where the order of effects matter you can't apply substitution based on referential transparency.
+This is, in general, true for effectful data types that are non-commutative. 
```

Comment:
I would add a new header here that reads:

`Effects with non-commutative monads`

Reply:
Also, the line should read `are commutative.`

Reply:
Addressed as part of the last master merge